### DISCUSSION TITLE
sensorname filter drop-down

### DISCUSSION THREAD

**Discussion by jianvector** [jasonish/evebox#298] [General]
If sensorname filter drop-down menu could be added, it would be a very good thing

![sensor](https://github.com/jasonish/evebox/assets/100907128/d6e20357-0e09-4a84-8b52-884d451dec0e)

**Comment by jianvector** [comment]
or how to do it?

**Reply by jasonish** [reply]
Should be easy to do, as I have the component in another view.. But my first attempt to add it messes with the flow of that row.. Will have to tinker with the CSS a little.

**Reply by jasonish** [reply]
I've added to this development build if you'd like to check it out.  The only thing keeping me from merging it to main is I'm not sure on the UI placement, but I do want it everywhere.

**Comment by jianvector** [comment]
I think it should at least appear on the alert/inbox/events/reports page,
or anywhere else

Jason Ish ***@***.***> 于2024年3月25日周一 07:26写道：

**Reply by jasonish** [reply]
Ah, I have it everywhere but the events page.

**Comment by jianvector** [comment]
Because evebox collects alerts for all sensors, sometimes it is necessary
to target a specific sensor when querying

myworld ***@***.***> 于2024年3月25日周一 09:14写道：

**Comment by jianvector** [comment]
Haha, thanks!

Jason Ish ***@***.***> 于2024年3月25日周一 10:09写道：
